components/SSI/SBTxWallet/index.tsx

```diff
@@ -69,57 +81,44 @@ function Component() {
         }
     }
 
+    const toggleActive = (id: string) => {
+        verifyArConnect(() => {})
```

Comment:
When rejecting, this verification is not prompted again (tho it could be still needed), let's revise.